unikraft/component/component.go

```diff
@@ -115,7 +115,7 @@ func urlHasVersion(u *url.URL) string {
 func parseStringProp(entry string) map[string]interface{} {
 	component := make(map[string]interface{})
 
-	if f, err := os.Stat(entry); err == nil && f.IsDir() {
+	if _, err := os.Stat(entry); err == nil {
```

Comment:
I'm fine with this, but at the same time if we want to keep the safety checks we can do

```
f.Mode().IsRegular() || f.Mode().IsDir()
```

Meaning we do not accept sockets/devices/other linux stuff